src/api/util.ts

```diff
@@ -29,3 +32,55 @@ export function toHex(num: number): string {
 export function isHex(possibleHexString: string): boolean {
   return /^0x[0-9a-fA-F]+$/.test(possibleHexString);
 }
+
+export function getBaseNftFromRaw(
+  rawBaseNft: RawBaseNft,
+  contractAddress: string
+): BaseNft {
+  return {
+    contract: { address: contractAddress },
+    tokenId: BigNumber.from(rawBaseNft.id.tokenId).toString(),
```

Comment:
nit: can you move the comments that were originally on the class over to the util as well?